Forte.ContentfulSchema.Tests/Core/ContentTypeDefinitionTests.cs

```diff
@@ -159,31 +131,69 @@ public void ShouldReturnFalseWhenUpdatingSameOrIncorrectEditorInterfaces(List<Ed
             }, 
         };
 
-        private EditorInterface CreateDefaultNewEditorInterface()
+        public static IEnumerable<object[]> ContentTypeModificators => new[]
+        {
+            new Action<ContentType>[] {ct => ct.SystemProperties.Id = "321"},
+            new Action<ContentType>[] {ct => ct.Name = "NewName"},
+            new Action<ContentType>[] {ct => ct.Description = "NewDescription"},
+            new Action<ContentType>[] {ct => ct.DisplayField = "NewDisplayField"}
+        };
+
+        public static IEnumerable<object[]> FieldsModificators => new[]
         {
-            var editor = new EditorInterface{Controls = new List<EditorInterfaceControl>()};
-            editor.Controls.AddRange(new List<EditorInterfaceControl>{ new EditorInterfaceControl { FieldId = "EditorField1", WidgetId = "NewWidgetId1" }, new EditorInterfaceControl { FieldId = "EditorField2", WidgetId = "NewWidgetId2" } });
-            
-            return editor;
-        }
+           new Action<List<Field>>[] { f =>
+                {
+                    f.Clear();
+                    f.Add(new Field {Id = "NewField1"});
+                }
+            },
+            new Action<List<Field>>[] { f => f[0].Name = "ChangedFieldName" },
+            new Action<List<Field>>[] { f => f[0].Type = "Text" },
+            new Action<List<Field>>[] { f => f[0].Omitted = true },
+            new Action<List<Field>>[] { f => f[0].Localized = true },
+            new Action<List<Field>>[] { f => f[0].Required = true },
+            new Action<List<Field>>[] { f => f[0].LinkType = "Entry" },
+            new Action<List<Field>>[] { f => f[0].Disabled = true },
+            new Action<List<Field>>[] { f => f[0].Validations = new List<IFieldValidator>(){new UniqueValidator()} },
+            new Action<List<Field>>[] { f => f[0].Items = new Schema(){LinkType = "TestLinkType"}},
+            new Action<List<Field>>[] { f => f[0].Items = new Schema(){Type = "TestType"}},
+            new Action<List<Field>>[] { f => f[0].Items = new Schema(){Validations = new List<IFieldValidator>(){new UniqueValidator()}}},
+            new Action<List<Field>>[] { f => f.Clear() },
+            new Action<List<Field>>[] { f => f.AddRange(new List<Field>{ new Field {Id = "NewField1"}, new Field {Id = "NewField2"} }) },
+        };
 
-//        [Theory]
-//        [InlineData(false)]
-//        [InlineData(true)]
-//        public void ShouldReturnValueDependingFromTheFieldComparerResult(bool areEqual)
-//        {
-//            var fieldComparer = new Mock<IEqualityComparer<Field>>();
-//            fieldComparer.Setup(m => m.Equals(It.IsAny<Field>(), It.IsAny<Field>()))
-//                .Returns(areEqual);
+        public static readonly Action<List<Field>> OrderFieldsModificators = f =>
```

Comment:
This does not seem to be needed anymore (if the reversal of fields order is added as a modification action above.